aspen/renderers.py

```diff
@@ -0,0 +1,5 @@
+
+# for backwards compatibility with aspen-renderer modules
```

Comment:
Baby steps. Let's not break backward compatibility just yet. Esp. Since we might be breaking simplates out entirely.